core/src/com/unciv/ui/civilopedia/CivilopediaText.kt

```diff
@@ -53,31 +91,93 @@ class FormattedLine (
         }
     }
 
+    /** Translates [centered] into [libGdx][Gdx] [Align] value */
+    val align
```

Comment:
If it's computed on-the-fly it should be a function to indicate that. Also this saves us a line.

fun align() = ...

Reply:
I think that was meant as "...let's see if caching is worth it..." -> do implementation -> forget while testing. Now calls _are_ repeated, so I'd prefer to complete the lazy. Even if the thing is cheap. I tend to see this whole system as potentially performance-critical, the worst offenders are still glaringly untouched, so cheap ones get a +1.